### ISSUE TITLE
Write script to clone code

### ISSUE THREAD

**Issue by tst-basilhuffman** [google/earthenterprise#540]
As a corollary to PR #475 a script should be written to clone code. It should handle both the case of cloning the main repo as well as a user-forked repo.

**Comment by tst-ppenev** [comment]
:stuck_out_tongue: There, actually, is such a script among the Docker scripts, too.

**Comment by tst-basilhuffman** [comment]
Is this pull-gee-repository.sh ?

**Comment by tst-ppenev** [comment]
Yes, it is.  Although, this one doesn't contain calculating the repository URL from a Git Hub username.  We could make it a bit more user-friendly, and get it out of the Docker scripts directory.

**Comment by tst-basilhuffman** [comment]
I can do this when time comes.
